### PR TITLE
Tokenization

### PR DESCRIPTION
**Pull request by turrence**
2 functions were created:

tokenize(data, metadata) -> returns token if the value.
There is an example usage in the comments of the code. While the metadata is not necessary, it would help readability if we are just looking at the database.

retrieveData(token, callback) -> within the callback(json_data) we can see the json_item retrieving the data itself and the metadata.
There is an example usage in the comments of the code.

### CODE REVIEW

File: cloud-haven-react-app/src/tokenizer.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+// https://stackoverflow.com/questions/5010288/how-to-make-a-function-wait-until-a-callback-has-been-called-using-node-js
```
Comment (glkidd) [inline cloud-haven-react-app/src/tokenizer.js:1]: I think we should probably take this out

File: dynamoDb.sh [hunk 2]
```diff
@@ -3,12 +3,34 @@
 
 # AttributeName=Password,AttributeType=S \
 
+# aws dynamodb create-table \
```
Comment (glkidd) [inline dynamoDb.sh:6]: Can we take these comments out?

File: dynamoDb.sh [hunk 3]
```diff
@@ -3,12 +3,34 @@
...
-        AttributeName=Username,AttributeType=S \
+        AttributeName=Token,AttributeType=S \
     --key-schema \
-        AttributeName=Username,KeyType=HASH \
+        AttributeName=Token,KeyType=HASH \
     --provisioned-throughput \
         ReadCapacityUnits=10,WriteCapacityUnits=5 \
     --endpoint-url http://localhost:8000
+
+# aws dynamodb create-table \
```
Comment (glkidd) [inline dynamoDb.sh:26]: And these comments?

**Review by glkidd — CHANGES_REQUESTED** [review]
Just commented on some comments I think we could take out.

File: cloud-haven-react-app/src/tokenizer.js (hunk 1 above)
Comment (turrence) [reply cloud-haven-react-app/src/tokenizer.js:1]: removed

File: dynamoDb.sh (hunk 2 above)
Comment (turrence) [reply dynamoDb.sh:6]: If there's a table that already exists, this causes an error, so I had it commented out. I'll leave it in and write a comment.

File: dynamoDb.sh (hunk 3 above)
Comment (turrence) [reply dynamoDb.sh:26]: This can be removed

**Review by glkidd — APPROVED** [review]

**Review by ShreyaTumma — APPROVED** [review]

**Pull request merged**